packages/lokse/src/commands/update.ts

```diff
@@ -53,6 +59,7 @@ class Update extends Base {
     const languages = flags.languages?.split(",") ?? this.conf?.languages;
     const column = flags.col ?? this.conf?.column;
     const format = flags.format ?? this.conf?.format ?? defaultFormat;
+    const sheets = flags.sheets?.split(",") ?? this.conf?.sheets ?? undefined;
```

Comment:
Is `?? undefined` at the end necessary if `this.conf?.sheets` gives `undefined` any way?

Reply:
Maybe in **this.conf?.sheets** could be some unwanted value?

Reply:
Thats true, we don't need  `undefined` here 👍

Check for valid filter value is on line 80 😉